doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +95,25 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
 
 The Cloud Services and the Cloud Resources Brokers provide value-added services in addition to the fundamental capabilities like service and resource discovery.  These Brokers are critical for a multi-cloud environment to function and utilise cloud specific plugins to perform the necessary activities. These Brokers can, for example, provision and manage environments with resources and services for Machine Learning (ML) services, Augmented/Virtual Reality, or specific industries.
 
+<a name="8.2.4"></a>
+### 8.2.4 Aspects of Multi-Cloud Security
+As cloud infrastructures become a key element in the telecommunication operator ecosystem, particularly with the 5G rollout becoming a critical business necessity, cloud-focused threats and associated adversarial behaviours, methods, tools, and strategies that cyber threat actors use to plan and execute are part of the attack surface landscape.
+ 
+In an ecosystem comprised of different network domains, products and business partnerships, the responsibility for managing these different cloud environments necessary to support 5G use cases falls to different organisations, creating a new level of complexities and a new range of security risks. In such an environment, there are additional security principles to be considered.  These principles, see Table 2 below, drawn from the collaboration with GSMA FASG.
+  
+| Multi-cloud Security Principle | Description |
+| ------------------------------ | ----------- |
+| Policy synchronization | Consistency in applying the right security policies across environments, services, interfaces, and configured resources |
+| Visibility | A common data model approach to capture events and behaviours across all the key compute, storage, network, and applications resources, environments, virtualised platforms, containers and interfaces |
+| Monitoring | Centralisation, correlation, and visualisation of security information across the different cloud environments to provide an end-to-end view and enable timely response to attacks |
```

Comment:
We may need to look at the networking model where instead of raw information shared by EMS to NMS abstractions are shared.

The timely response to individual instances still falls to the affected enterprise.  The central correlation can point to an attack on multiple enterprises with the intent of disabling or comprising the whole.

Not sure how to capture this.

Reply:
@pgoyal01:I am not sure that this fits into this section. Can we have a separate discussion, perhaps in wiki on this?